Bangladesh OCDID Update

NaviThotlu opened the pull request:
Hi @Zohour,

1. Added Bangladesh new constituencies in country-bd.csv
2. Added Bangladesh new constituencies in Jatiya_Sangsad_electorates.csv
3. Added README file.

asmitajha91 commented:
@azuser Hi Zohour, could you please help review this PR.

HKSenior commented on identifiers/country-bd/Jatiya_Sangsad_electorates.csv:
According to this [wiki](https://en.wikipedia.org/wiki/List_of_constituencies_of_the_Jatiya_Sangsad) Jatiya Sangsad has 8 divisions. The following districts aren't represented within the additions, please add them:
- Barishal of the Barisal district, 
- Jhalokati of the Barisal district
- Moulivibazar of the Sylhet district

HKSenior commented on identifiers/country-bd/README:
```diff
@@ -0,0 +1,9 @@
+File contents in `country-bd` directory:
+
+* Jatiya_Sangsad_electorates.csv:contains the list of electoral districts for the House of the Nation (Jatiya Sangsad) of Bangladesh
+
```
nit: remove new line from between list elements.

HKSenior commented on identifiers/country-bd/README:
```diff
@@ -0,0 +1,9 @@
+File contents in `country-bd` directory:
+
+* Jatiya_Sangsad_electorates.csv:contains the list of electoral districts for the House of the Nation (Jatiya Sangsad) of Bangladesh
```
nit: add a space after the colon.

```suggestion
* Jatiya_Sangsad_electorates.csv: contains the list of electoral districts for the House of the Nation (Jatiya Sangsad) of Bangladesh
```

NaviThotlu commented:
Hi @HKSenior,

1. We have updated Barisal to Barishal, but for Jhalokati and Moulivibazar we have checked those have names as Jhalokathi and Maulvibazar in the Bangladesh parliament website. Please refer to screen shot below.
![image001](https://github.com/opencivicdata/ocd-division-ids/assets/134617739/66d0fda3-2b09-4a28-a681-a199eee20bff)

2.  we have updated README file with the changes mentioned.

HKSenior commented on identifiers/country-bd/Jatiya_Sangsad_electorates.csv:
```diff
@@ -0,0 +1,301 @@
+ocd-division/country:bd/ed:rangamati,Rangamati
+ocd-division/country:bd/ed:bandarban,Bandarban
+ocd-division/country:bd/ed:khagrachari,Khagrachari
+ocd-division/country:bd/ed:coxs_bazar_1,Cox s Bazar-1
```
It's okay to have an apostrophe within the name column. Please add here and the following lines.

```suggestion
ocd-division/country:bd/ed:coxs_bazar_1,Cox's Bazar-1
```

Screenshot from website:
![Screenshot 2023-10-25 at 8 10 20 AM](https://github.com/opencivicdata/ocd-division-ids/assets/10860174/fa6d48c2-2425-4610-bd16-e27e5db6cfaf)

HKSenior reviewed (changes requested):
Thanks for the clarification on the previous divisions.

NaviThotlu commented:
Hi @HKSenior ,

we have added apostrophe in Cox's Bazar.

jloutsenhizer reviewed (approved):
Did a quick spot check, everything looks good to me.

jpmckinney reviewed (approved):
Looks good. Makes sense to omit the division and district from the OCD-ID, and just keep the constituency. (I quickly compared to https://en.wikipedia.org/wiki/List_of_constituencies_of_the_Jatiya_Sangsad)
